### ISSUE TITLE
gatsby-remark-prismjs HTML code rendered within codeblock

### ISSUE THREAD

**Issue by Undistraction** [gatsbyjs/gatsby#5311]
## Description

HTML inside code-block is rendered when no language is defined on the codeblock.

### Steps to reproduce

Add a codeblock to a markdown doc containing, for example an iframe:

```
<iframe src="https://www.youtube.com/embed/v30Ml-9xsRQ" frameborder="0" allowfullscreen></iframe>
&#96;
```

### Expected result

The  html code is displayed as text within the codeblock.

### Actual result

The iframe is rendered within the codeblock.

### Workaround

Add a language to the codeblock:
```
```html
<iframe src="https://www.youtube.com/embed/v30Ml-9xsRQ" frameborder="0" allowfullscreen></iframe>
&#96;
```

### Notes

This doesn't happen unless `gatsby-remark-prismjs` is installed. 

### Environment

* Gatsby version  ^1.9.259
* Gatsby Remark Prism version ^2.0.2
* Node.js version: 9.3.0
* Operating System: OSX 10.11.6

**Comment by KyleAMathews** [comment]
Hmm, that's not good. Could you investigate what's happening? https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-remark-prismjs/src/index.js

**Comment by docwhat** [comment]
Huh, I thought I fixed that in PR #4620 — Only if you specify the language as `none` should it show the `<iframe>` verbatim; otherwise it should always escape.
